docs/includes/apiargs-MongoDBCollection-common-option.yaml

```diff
@@ -42,6 +42,12 @@ description: |
 
    This is not supported for server versions prior to 3.2 and will result in an
    exception at execution time if used.
+
+   It is not possible to specify a :manual:`read concern
+   </reference/read-concern>` for individual operations as part of a
+   transaction. Instead, set the :manual:`read concern
```

Comment:
We've already linked to the read concern manual entry, so I think this bit can read "set the ``readConcern`` option" so that we refer to the actual option name for `startTransaction()`.